dev/ansible/04_upload_files.yml

```diff
@@ -0,0 +1,67 @@
+- name: set variable
+  set_fact:
+    consul_token: "{{ lookup('env', 'consul_master_token') }}"
+  tags:
+    - example-upload
+    - test
+
+# https://unix.stackexchange.com/a/148698
+- name: Start local proxy on port 9999 to minio service
+  shell: nohup consul connect proxy -service minio-local -upstream minio:9999 -log-level debug </dev/null >/dev/null 2>&1 &
+  when: not lookup('env', 'consul_acl') | bool or (lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'allow')
+  tags:
+    - example-upload
+    - test
+
+# https://unix.stackexchange.com/a/148698
+- name: Start local proxy on port 9999 to minio service with token
+  shell: nohup consul connect proxy -service minio-local -upstream minio:9999 -log-level debug </dev/null >/dev/null 2>&1 &
+  when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
+  environment:
+    CONSUL_HTTP_TOKEN: "{{ consul_token }}"
+  tags:
+    - example-upload
+    - test
+
+# To quickly verify that proxy available (without any credentials)
+# https://github.com/minio/minio/issues/4851#issuecomment-324543717
+- name: Minio curl example. Verify that proxy to minio available
+  uri:
+    url: http://127.0.0.1:9999
+    status_code: 403
+  register: curl_output
+  until: curl_output.status == 403
```

Comment:
Why is the status `403`?

Reply:
lack of intention. Fixed in https://github.com/fredrikhgrelland/terraform-nomad-hive/pull/39/commits/34f1c8341321e50f01bf1707dc188f5c2f9d079d

Reply:
Ahh, got it! 👍